index.html

```diff
@@ -9,21 +9,64 @@
   </head>
   <body>
     <header>
-      <h1>Awesome books</h1>
-      <div id="books-container"></div>
+      <nav class="navbar">
+        <div class="nav-branding">Awesome Books</div>
+
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="list">List</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="add-new">Add New</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="contact">Contact</a>
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly add the vertical bars that separate the nav items so that your design matches the provided wireframe.

**How your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/183020991-3dd8c0ff-9ef7-4774-aa7a-df3e26eff7de.png)

**The desired design from the wireframe:**

![image](https://user-images.githubusercontent.com/86318284/183021100-8e6391b9-d2ef-4752-b198-560528c8edf1.png)

Reply:
@Nemwel-Boniface Thank you for this review. I will get it done now.👍